modules/custom/reserve_movies/js/ShowMoviesByGenre.js

```diff
@@ -0,0 +1,18 @@
+function ShowMoviesByGenre() {
+    window.onload = function() {
+        var movieGenre = localStorage.getItem("movieGenre");  
+        $('#genre').val(movieGenre);
+    }
+    $("#genre").on("change", function () {
+        var genre = $(this).val();
```

Comment:
Mora li var?

Reply:
Ne mora, s obzirom na to da je ona ili globalnog scopea ili scopea funkcije , za potrebe ove funkcionalnosti trebalo bi da koristim let, posto moze da bude izmenjena a njen block scope mi je dovoljan za potrebe ovog zadatka.